Documentation/network/concepts/ipam/multi-pool.rst

```diff
@@ -137,6 +137,13 @@ taken from the ``ipam-multi-pool-pre-allocation`` map.
 
  .. _ipam_crd_multi_pool_limitations:
 
+Routing to Allocated PodCIDRs
+-----------------------------
+PodCIDRs allocated from ``CiliumPodIPPools`` can be announced to the network by the
+:ref:`bgp_control_plane` (:ref:`bgp_control_plane_multipool_ipam`). Alternatively,
+the ``auto-direct-node-routes`` Helm option can be used to enable automatic routing
```

Comment:
`auto-direct-node-routes` looks like an agent flag not a Helm value. Could you either refer to it as an agent flag or mention the Helm value here?